interfaces/opensearch_client/v0/schemas/provider.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://canonical.github.io/charm-relation-interfaces/interfaces/opensearch_client/schemas/provider.json",
+    "title": "`opensearch_client` provider schema",
+    "description": "The `opensearch_client` root schema comprises the entire provider databag for this interface.",
+    "type": "object",
+    "default": {},
+    "required": [
+        "username",
+        "password",
+        "endpoints"
+    ],
+    "additionalProperties": true,
+    "properties": {
+        "username": {
+            "$id": "#/properties/username",
+            "title": "Relation User Username",
+            "description": "Username for connecting to the requested index",
+            "type": "string",
+            "default": "",
+            "examples": [
+                "opensearch-client_0_user"
+            ]
+        },
+        "password": {
+            "$id": "#/properties/password",
+            "title": "Relation User Password",
+            "description": "Password for connecting to the requested index",
+            "type": "string",
+            "default": "",
+            "examples": [
+                "alphanum-32byte-random"
+            ]
+        },
+        "endpoints": {
+            "$id": "#/properties/endpoints",
+            "title": "Relation endpoints",
+            "description": "A list of endpoints used to connect to the index",
+            "type": "string",
+            "default": "",
+            "examples": [
+                "unit-1:port,unit-2:port"
```

Comment:
The example should be an actual example, `port` wouldn't work for a port number.